tests/xcm-transfer/shared.ts

```diff
@@ -138,7 +139,11 @@ export default function buildTest(tests: ReadonlyArray<TestType>) {
           if ('checkUmp' in testOpt) {
             await checkUmp(fromChain).toMatchSnapshot('from chain ump messages')
           } else {
-            await checkHrmp(fromChain).toMatchSnapshot('from chain hrmp messages')
+            await checkHrmp(fromChain)
+              .map((v) => JSON.parse(JSON.stringify(v)))
+              // redact setTopic
+              .map((v) => _.update(v, '[0].data[1].v3[4].setTopic', () => 'redacted'))
```

Comment:
we already have redact hex

Reply:
I want to redact only setTopic which is something thats changes overtime. This is a workaround. Proper solution will be to add a filter method on expect

Reply:
https://github.com/AcalaNetwork/chopsticks/pull/554/files#diff-7f14b5bccef1ce97227b2ea7bb4dfed35262eb8c0cbb9f4c98b513f1a7d253d4R21

@xlc added option on chopsticks-testing